divviup/build.gradle.kts

```diff
@@ -103,6 +103,28 @@ publishing {
             afterEvaluate {
                 from(components["release"])
             }
+
+            pom {
+                name.set("divviup-android")
```

Comment:
FWIW newer versions of Gradle would support `name = "divviup-android"` assignment syntax here, but `.set()` works fine as-is.